.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,22 @@
+## For issue
+
 ## Please verify the following:
 
 - [ ] `yarn test` **jest** tests pass with new tests, if relevant
 - [ ] `yarn lint` **eslint** checks pass with new code, if relevant
 - [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
 - [ ] `README.md` (or relevant documentation) has been updated with your changes
+- [ ] Include issue number in title and description
+- [ ] (If applicable) The feature related to this PR was tested across various use cases to ensure it works correctly.
```

Comment:
I think if we get a little more specific this will be more clear, something like:

```suggestion
- [ ] If this affects functionality there aren't tests for, I manually tested it, including by generating a new app locally if needed ([see docs](https://docs.infinite.red/ignite-cli/contributing/Contributing-To-Ignite/#testing-changes-from-your-local-copy-of-ignite)).
```